### ISSUE TITLE
The Southbourne Tax Group: Get richer this tax season in 10 easy ways millennials can maximize refunds and avoid costly mistakes

### ISSUE THREAD

**Issue by mcleanxandra** [mcleanxandra/The-Southbourne-Tax-Group#5]
By now, you've probably heard people talking about tax season — or maybe you got the hint that you'll need to start filing your taxes soon, because of that complicated-looking W-2 form that arrived at your house.

It's only natural to want to procrastinate on filling out those ugly forms (1040-what?) or to feel intimidated or overwhelmed.

Not only do most millennials say they fear filing their taxes — more so than other generations — but young Americans are also scared to make a mistake on their taxes, and are less likely to seek <a href="http://blurpalicious.com/the%20southbourne%20tax%20group%20business%20accounting%20services%20hong%20kong">professional guidance</a>.

That might be especially true for you if this year is only your first (or second... or third...) time giving it a try without help from parents.

In fact, you should remember that tax time can be a happy time — especially if you are due for a big refund in the form of a check from the government.

Here are some pointers on what to look out for as you're doing your taxes, so you avoid mistakes and maximize any money due to you. And remember: When in doubt, you can always call the free tax help line that the IRS provides.

<b>1. Know the deadlines — or pay the price</b>

First and foremost, you simply must review a simple list (like this one from Mic) showing when taxes are due. Don't be off, even by a couple of days, or else you run the risk of paying a penalty for filing late.

You might want to circle April 18 on your calendar — as that’s the drop-dead deadline to file your taxes this year. And if you haven’t received your W-2 from your employer by Valentine’s Day, you might want to ring the IRS for help or to file an extension.

Speaking of extensions…

<b>2.Don't make this common mistake regarding extensions</b>

Time waits for no man, and apparently, neither does Uncle Sam.

There might be a reason why you choose to file for a tax extension (hopefully you get it!) that could give you six months of breathing room to get your ducks and tax paperwork in a row.

That, however, does not mean you get to simply shoot an IOU to the government, should you owe them. According to the IRS, "an extension will give you extra time to get your paperwork to the IRS, but it does not extend the time you have to pay any tax due."

"It is a common misconception that if you file for an extension, your obligation to pay your tax bill is also delayed. This is not true!" Jacob Dayan, a partner and co-founder of Community Tax, a tax resolution and debt relief services company, reiterated to Mic. "If you don't pay your tax bill at the deadline you are delinquent, even if you've requested an extension for your return."

<b>3.Beware tax-related identity theft</b>

You might be dreaming about all the awesome things you plan to buy with the tax return you hope to get, and we hope you’re able to enjoy it!

But one common danger all millennials (and everyone, really) needs to look out for is tax-related identity theft. Believe it or not, there are cold-hearted people out there who are looking for opportunities to steal your Social Security number — in order to file a phony tax return and claim your bounty.

In 2016 alone, the IRS discovered thousands of fraudulent refunds and has been working on the double to reduce tax return fraud.

Alex Hamilton, a communications professional at the Identity Theft Resource Center, advises taxpayers to file as early as possible to help reduce the risk of tax-related identity theft. In addition, Hamilton told Mic, it's important to "regularly update your anti-virus software to protect you from a cyber-attack which can steal your personal data."

To prevent tax-related identity theft, the IRS also encourages tax filers to question suspicious emails and "threatening calls" from people posing as representatives of your bank — or even the IRS — and not to carry your Social Security card.

For more tips on how to protect yourself from tax-refund fraud, visit the IRS website.

<b>4.Use student loans — to your advantage</b>

Sadly, it feels like student loan debt is just a universal part of being a millennial.

But there's a silver lining: Millennials contending with student loans have the opportunity to deduct interest paid on student loans.

"Many millennials rush to get their taxes done online as soon as they get their W-2s and don’t wait for their 1098-Es from their student loan providers," Joseph Carpenito, a licensed <a href="http://www.demigodfrey.n.nu/the-southbourne-tax-group-accounting-for-half-truths">financial advisor</a> at Raymond James and founder of the financial site MyPlan2Day.com, explained to Mic. "[For] a young person with limited deductions, student loan interest may potentially be one of their largest deductions."

You can deduct up to $2,500 of student loan interest paid in a given year.

There are, however, income limits for this deduction, so be sure to check out tips for claiming the student loan interest deduction on the IRS website. 

<b>5.Make the most of small business tax perks if you work a side hustle</b>

Do you love moonlighting as a freelancer or someone who earns extra money doing what you love? Surprise!

The government might actually consider you to be a small business owner.

"Millennials are more involved in the 'gig economy' than other demographics, and many of them like the appeal of working for themselves in jobs like Uber drivers and delivery drivers," Max Robinson, an associate for Jumpstart, a research and experimentation tax credit specialist company, told Mic. 

"However, these millennials need to realize that working in jobs like this classifies them as small business owners. This means they're eligible for certain deductions, like petrol expenses," Robinson added.

TL;DR: You might be owed cash back if you spent money on business expenses, so read up on what might count. Cha-ching!

<b>6.Know what the 1099-MISC is for</b>

As the IRS mentions on their website, you don’t have to have a business in order to report income as self-employed or an independent contractor on your taxes. 

“In most cases, if you receive a form 1099-MISC it means you are considered self-employed by the government,” Crystal Stranger, president of the tax firm 1st Tax and author of The Small Business Tax Guide, told Mic. 

“Not only does this mean you need to report your income and expenses on a Schedule C and pay self-employment (Social Security & Medicare) taxes on your net income, but also, in a lot of places, you will need to register with the city or state and may owe taxes or at least have a filing requirement on the local level," Stranger said. "Missing the local level filing can have bigger tax and penalty implications than making a mistake on your federal taxes.”

<b>7.Use tax software to save time (and money)</b>

Filing your taxes can be a truly overwhelming experience — especially if it’s your first time. It’s easy to feel stressed and worried you're not qualified to operate in the land of adulthood without a parent or guardian nearby.

But that doesn't mean you can't take care of business yourself.

"This may be your first time doing your own taxes, but you don’t have to take your taxes somewhere and pay hundreds of dollars to have them prepared," TurboTax Certified Public Accountant Lisa Greene-Lewis told Mic. "You most likely are one of the 60 million Americans who has a relatively straightforward tax return, so there is no reason to pay someone to do your taxes. You may even be able to file your federal taxes for free."

Companies like TurboTax, H&R Block, and even the IRS offer free tax-filing services to qualifying taxpayers that will not only make your life a little more hassle free, but also help keep some money in your pocket.

Just remember to make sure to report business expenses and other opportunities for deductions when the software prompts you. Speaking of which...

<b>8.Dig deep for deductions</b>

In a rush to get through your taxes as quickly — and with as few gray hairs — as possible, it becomes easy to skip over deductions that can help lower your tax bill. In fact, there are a number of overlooked tax deductions that some experts consider money left on the table.

“It’s easy to overlook the many ways you stay fabulous by helping others throughout the year,” Richard Lavina, CEO of Taxfyle, a personal and business tax filing app, told Mic.
